dsl/src/main/scala/org/http4s/dsl/impl/Path.scala

```diff
@@ -166,10 +166,13 @@ object /: {
     }
 }
 
-protected class PathVar[A](cast: String => Try[A]) {
+protected class PathVar[A](cast: String => Option[A]) {
+  def this(cast: String => Try[A])(implicit ev: DummyImplicit) =
```

Comment:
Had ideas to mark this deprecated but not sure. Thoughts?

Reply:
Yes.  It's a suboptimal implementation.